R/mboostLSS.R

```diff
@@ -73,7 +73,8 @@ blackboostLSS <- function(formula, data = list(), families = GaussianLSS(),
 mboostLSS_fit <- function(formula, data = list(), families = GaussianLSS(),
                           control = boost_control(), weights = NULL,
                           fun = mboost, funchar = "mboost", call = NULL,
-                          method, ...){
+                          method, 
+                          timeformula = NULL, ...){
```

Comment:
Is it possible to specify `timeformula` to `...` and extract it later e.g. with `dots <- as.list(...); timeformula <- dots["timeformula"]`?`I do not want to change the interface just for FDboost